### ISSUE TITLE
Kernel update request -DSD pop noise fix

### ISSUE THREAD

**Issue by epicfrequency** [volumio/platform-asus#25]
Per backport in Tinkerboard's debian kernel repo. Request to update the fix into Volumio tinker board image as well 

https://github.com/TinkerBoard/debian_kernel/commit/44e2ce5d21774feaf90ba428db695a62ee9adb41

Eliminate noise at the start of DSD playback.
ALSA: usb-audio: Eliminate noise at the start of DSD playback.
Copy mainline DSD playback pop noise fix

torvalds@0120073#diff-cc9fc192670bbec31122ad6874641755

[Problem]
In some USB DACs, a terrible pop noise comes to be heard
at the start of DSD playback (in the following situations).

- play first DSD track
- change from PCM track to DSD track
- change from DSD64 track to DSD128 track (and etc...)
- seek DSD track
- Fast-Forward/Rewind DSD track

[Cause]
At the start of playback, there is a little silence.
The silence bit pattern "0x69" is required on DSD mode,
but it is not like that.

[Solution]
This patch adds DSD silence pattern to the endpoint settings.

Signed-off-by: Nobutaka Okabe <nob77413@gmail.com>
Signed-off-by: Takashi Iwai <tiwai@suse.de>

**Comment by volumio** [comment]
Thanks, we'll look at it.
@gkkpch what do you think?

**Comment by gkkpch** [comment]
Yes, I saw that last Friday. It's already done in our volumio version of the tinkerboard kernel. (This is  temporary solutuion, we will eventually change this back to the official Asus after Michelangelo and I have tested & verified the Rockchip fix for the sampling frequency issue. 
As this will take a while, we will update our own kernel so a new Image can be built (approx. 2 weeks from now)
